### PR TITLE
[CSL-1145] Demolish MonadStats

### PR DESCRIPTION
**Pull request by gromakovsky**
Paket.

### CODE REVIEW

File: src/Pos/Launcher/Launcher.hs [hunk 1]
```diff
@@ -33,37 +31,24 @@ import           Pos.WorkMode               (ProductionMode, StaticMode, StatsMo
 runNodeProduction
     :: forall ssc.
        (SscConstraint ssc, SecurityWorkersClass ssc)
-    => Transport (ProductionMode ssc)
-    -> KademliaDHTInstance
+    => KademliaDHTInstance
```
Comment (pva701) [inline src/Pos/Launcher/Launcher.hs]: lol

File: src/Pos/Launcher/Launcher.hs [hunk 2]
```diff
@@ -33,37 +31,24 @@ import           Pos.WorkMode               (ProductionMode, StaticMode, StatsMo
...
+runNodeProduction kinst transport plugins np sscnp =
+    runProductionMode kinst transport np sscnp (runNode @ssc plugins)
 
 -- | Run full node in static mode
 runNodeStatic
     :: forall ssc.
        (SscConstraint ssc, SecurityWorkersClass ssc)
-    => Transport (StaticMode ssc)
-    -> Set NodeId
+    => Set NodeId
```
Comment (pva701) [inline src/Pos/Launcher/Launcher.hs]: zachem???

File: src/Pos/Statistics/StatEntry.hs [hunk 3]
```diff
@@ -1,66 +0,0 @@
...
-instance Monoid ValueStat where
-    mappend a b = ValueStat
-        { valueCount = valueCount a + valueCount b
-        , valueMin = valueMin a `min` valueMin b
-        , valueMax = valueMax a `max` valueMax b
-        , valueSum = valueSum a + valueSum b
-        }
-    mempty = ValueStat
-        { valueCount = 0
-        , valueMin = 1.0/0.0     -- Infinity
```
Comment (pva701) [inline src/Pos/Statistics/StatEntry.hs]: ahahah

File: src/Pos/Launcher/Launcher.hs (hunk 2 above)
Comment (gromakovsky) [inline src/Pos/Launcher/Launcher.hs]: Because `Transport` is `Transport`.

**Pull request merged**